src/hermez-account.js

```diff
@@ -81,4 +83,15 @@ module.exports = class HermezAccount {
         tx.fromAx = this.ax;
         tx.fromAy = this.ay;
     }
+
+    /**
+     * Sign withdraw-bjj
+     * @param {Object} inputs - Zk inputs  for the withdraw-bjj circuit
+     * @return {Scalar} Signature parameters.
```

Comment:
```suggestion
     * @return {Scalar} Signature parameters
```

Reply:
signature is retirned as an Object with `{ R8, S }`.
I would change `Scalar` for `Object`